docker-compose.dev.yml

```diff
@@ -1,12 +1,13 @@
-version: '3.7'
 services:
   traefik:
     image: traefik:v2.10
     container_name: traefik
     restart: unless-stopped
+    extra_hosts:
+    - "host.docker.internal:host-gateway"
     ports:
-      - "80:80"
-      - "443:443"
+      - "127.0.0.1:80:80"
+      - "127.0.0.1:443:443"
```

Comment:
comment: this forces docker to use IPv4 when using this ports since it can "confuse" between IPv4 and IPv6